packages/mdc-animation/README.md

```diff
@@ -27,59 +27,68 @@ npm install --save @material/animation
-| `$mdc-animation-acceleration-curve-timing-function` | `cubic-bezier(.4, 0, 1, 1)` | Animations that cause objects to leave the screen permanently (e.g. a fade out) |
-| `$mdc-animation-sharp-curve-timing-function` | `cubic-bezier(.4, 0, .6, 1)` | Animations that cause objects to leave the screen temporarily (e.g. closing a drawer) |
+CSS Class | Description
+--- | ---
+`mdc-animation-deceleration-curve` | Sets the speed curve to deceleration
+`mdc-animation-standard-curve` | Sets the speed curve to standard, a.k.a quickly accelerate and slowly decelerate
+`mdc-animation-acceleration-curve` | Sets the speed curve to acceleration
+`mdc-animation-sharp-curve` | Sets the speed curve to sharp, a.k.a quickly accelerate and decelerate
 
-### SCSS
+### Sass Variables and Mixins
 
-Simply drop `mdc-animation` into your build and start using the variables:
+Instead of setting CSS classes on elements, you can use the Sass mixins to achieve the same goal.
 
 ```scss
-.mdc-thing--animating {
-  animation: foo 175ms $mdc-animation-standard-curve-timing-function;
+.my-element--animating {
+  @include mdc-animation-acceleration-curve;
 }
 ```
 
-or the mixins, which simply assign their corresponding variables to the `animation-timing-function`
-property:
+Mixin | Description
+--- | ---
+`mdc-animation-deceleration-curve` | Sets the speed curve to deceleration
+`mdc-animation-standard-curve` | Sets the speed curve to standard, a.k.a quickly accelerate and slowly decelerate
+`mdc-animation-acceleration-curve` | Sets the speed curve to acceleration
+`mdc-animation-sharp-curve` | Sets the speed curve to sharp, a.k.a quickly accelerate and decelerate
+
+We also provide the timing functions used by these mixins, which you can use with the `animation` or `transition` CSS properties
 
 ```scss
-.mdc-thing--on-screen {
-  @include mdc-animation-acceleration-curve;
+.my-element--animating {
+  animation: foo-keyframe 175ms $mdc-animation-standard-curve-timing-function;
 }
 ```
 
-Every mixin has the same name as its corresponding variable, without the `-timing-function` suffix.
+Variable | Description
+--- | ---
+`mdc-animation-deceleration-curve-timing-function` | Timing function to decelerate
+`mdc-animation-standard-curve-timing-function` | Timing function to quickly accelerate and slowly decelerate
+`mdc-animation-acceleration-curve-timing-function` | Timing function to accelerate
+`mdc-animation-sharp-curve-timing-function` | Timing function to quickly accelerate and decelerate
 
-MDC Animation also provides helper functions for defining transitions for when something enters and exits the frame. A
-very common example of this is something that fades in and then fades out using opacity.
+The following functions create transitions given `$name` and the `$duration`. You can also specify `$delay`, but the default is 0ms. `$name` can either refer to the keyframe, or to CSS property used in `transition`.
 
 ```scss
-@import "@material/animation/functions";
```

Comment:
It might be easier for newcomers if we keep the `@import` declarations (and add them to the examples that lack them) so they don't have to spend time figuring which path to import.